.mergify.yml

```diff
@@ -4,22 +4,51 @@ defaults:
     # mergify.io has removed bot_account from its free open source plan.
     comment:
     # bot_account: ceph-csi-bot # mergify[bot] will be commenting.
-    merge:
+    queue:
       # merge_bot_account: ceph-csi-bot #mergify[bot] will be merging prs.
       # update_bot_account: ceph-csi-bot #mergify will randomly pick and use
       # credentials of users with write access to repo to rebase prs.
+      name: default
       method: rebase
       rebase_fallback: merge
-      strict: smart
-      strict_method: rebase
-    rebase:
-    # bot_account: ceph-csi-bot # same as update_bot_account.
-
+      update_method: rebase
+queue_rules:
```

Comment:
this might not work for the release-3.4 branch as we don't run k8s-e2e-external-storage tests on it and this won't work for the centos/ci branch as we don't run any of these tests on it.

Reply:
as @Rakshith-R  mentioned, centos ci branch need changes. will wait for the PR refresh, it should be covered in it.